app/src/main/java/org/fossasia/openevent/utils/SaveFilters.java

```diff
@@ -0,0 +1,41 @@
+package org.fossasia.openevent.utils;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.util.Log;
+
+/**
+ * Created by user on 3/1/2016.
+ */
+public class SaveFilters {
+
+    public boolean saveArray(Boolean[] array, String arrayName, Context mContext) {
+        SharedPreferences prefs = mContext.getSharedPreferences("Filters", 0);
+        SharedPreferences.Editor editor = prefs.edit();
+        editor.putInt(arrayName +"_size", array.length);
+        for(int i=0;i<array.length;i++)
+            editor.putBoolean(arrayName + "_" + i, array[i]);
+
+        return editor.commit();
+    }
```

Comment:
Suppose we are viewing the sessions of a particular track and we set the filters. Now we want to change alter the filters, so when we again open the filter dialog, the previous filters should be there. So we want to save the filters tiil we are in the Activity.